src/vectorchord/usage/search.md

```diff
@@ -0,0 +1,82 @@
+    - Type: Float
+    - Default: `1.9`
+    - Example:
+        - `SET vchordrq.epsilon = 0.1` means you not need a very high accuracy. 
+        - `SET vchordrq.epsilon = 4.0` means you need a very high accuracy.
+    - Note: The default value is `1.9`. The acceptable range is from `0` to `4`.
+
+    You can refer to [performance tuning](../usage/performance-tuning#query-performance) for more information about tuning the query performance.
+- `vchordrq.prewarm_dim`
+    - Description: The `vchordrq.prewarm_dim` GUC parameter is used to prewarm the projection matrix for the specified dimensions. This can help to reduce the latency of the first query after the database is restarted. The default value is `64,128,256,384,512,768,1024,1536`, which means that the projection matrix will be prewarmed for these dimensions.
+    - Type: String
+    - Default: `64,128,256,384,512,768,1024,1536`
+    - Example:
+        - `ALTER SYSTEM SET vchordrq.prewarm_dim = '64,128'` means that the projection matrix will be prewarmed for dimensions 64 and 128.
+    - Note: This setting requires a database restart to take effect.
+- `vchordrq.io_rerank`
+    - Description: This GUC parameter controls the I/O prefetching strategy for vector search, which can significantly impact search performance on disk-based vectors.
+    - Type: String
+    - Default: Depends on PostgreSQL version
+        - PostgreSQL 13, 14, 15, 16: `prefetch_buffer`
+        - PostgreSQL 17: `read_stream`
+    - Possible values:
+        - `read_buffer`: Indicates a preference for ReadBuffer.
+        - `prefetch_buffer`: Indicates a preference for both PrefetchBuffer and ReadBuffer. This option is optimized for disk vector search and is the default on PostgreSQL 13, 14, 15, 16.
+        - `read_stream`: Indicates a preference for read_stream. This option is optimized for disk vector search and is only available in PostgreSQL 17. It's the default on PostgreSQL 17.
+    - Notes:
+        - The prefetching distance of `prefetch_buffer` depends on the compile-time constant `WINDOW_SIZE`.
+        - The prefetching distance of `read_stream` depends on the GUC parameter `effective_io_concurrency`.
+        - Prefetching for the bit vector has not been implemented.
+        - Prefetching for the centroid has been implemented, but is currently disabled.
+    - Example:
+        - `SET vchordrq.io_rerank = 'prefetch_buffer'` to optimize for disk-based vector search on PostgreSQL versions before 17.
+        - `SET vchordrq.io_rerank = 'read_stream'` to use the optimized strategy available on PostgreSQL 17.
+- `vchordrq.early_filter`
```

Comment:
```suggestion
- `vchordrq.prererank_filtering`
```

Reply:
I think this option is not fully implemented yet, so we can leave it out for now.